Makefile

```diff
@@ -358,8 +361,26 @@ else
             exit $$rc
 endif
 
-release-tests:
-	$(Q): # An empty placeholder
+release-tests: e2e-tests
+
+e2e-tests: build
+ifndef WHAT
+	$(Q)what=test/e2e/policies; \
```

Comment:
Hmm, instead of `ifndef` could you use something like this:
```
$(Q)what=$(if $(WHAT),$(WHAT),"test/e2e/policies"); \
```
so that you wouldn't need to duplicate the rest of the script

Reply:
I'll change it like that. Much better.